src/app/routes/api/gmail.py

```diff
@@ -0,0 +1,134 @@
+"""
+This file contains the GMAIL API authorization and automated email processing for 
+the Service Account
+"""
+from __future__ import print_function
+from googleapiclient.discovery import build
+from apiclient import errors
+from email.mime.text import MIMEText
+import pybase64
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+
+
+# Email sending confirmation email
+EMAIL_FROM = "admin@piecemealapp2.com"
```

Comment:
This should not be visible in the source code. I'd recommend making it an environment variable.